hack/test-cmd.sh

```diff
@@ -865,6 +866,33 @@ __EOF__
   kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.unschedulable}}" '<no value>'
 
 
+  #########################
+  # Experimental resource #
+  #########################
+
+  ### Create and delete an experimental deployment example
+  # Pre-condition: no persistent experimental deployment currently exist
+  kube::test::get_object_assert experimental/deployment "{{range.items}}{{$id_field}}:{{end}}" ''
+  # Command
+  kubectl create -f examples/experimental/deployment.yaml
+  # Post-condition: deployment "nginx-deployment" is created
+  kube::test::get_object_assert experimental/deployment "{{range.items}}{{$id_field}}:{{end}}" 'nginx-deployment:'
+  # Clean up
+  kubectl delete experimental/deployment nginx-deployment
+  ### Get experimental resource via shortname
+  kubectl get experimental/hpa
+  ### Get experimental resource via shortname implicitly should fail
+  ERROR_FILE="/tmp/should-error"
```

Comment:
Quinton pointed this out: https://github.com/kubernetes/kubernetes/pull/15325#discussion_r41575869

"If multiple of these run in parallel, they will trample on each other. You need to use a unique file name here."
"Actually ensure-temp-dir below might be what you're looking for."